doc/tutorial/didi/carprice_xgboost/README.md

```diff
@@ -1,32 +1,35 @@
-# XGBoost on SQLFlow Tutorial
+# Predict the Car Price using XGBoost in SQLFlow
+
+This is a tutorial on how to train/predict an XGBoost model to fit the [Cars Dataset](https://www.kaggle.com/CooperUnion/cardataset). You can find more SQLFlow usage from the [User Guide](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/user_guide.md). In this tutorial, you will learn how to:
 
-This is a tutorial on a train/predict XGBoost model in SQLFLow, you can find more SQLFlow usage from the [User Guide](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/user_guide.md). In this tutorial, we will introduce the following:
 - Train an XGBoost model to fit the cars dataset.
-- predict the price of a car using a well-trained model.
-- Use the [SHAP EXPLAINER](https://github.com/slundberg/shap) tool to interpret the output of the trained model to known how these features affect the price.
+- Predict the car price using a well-trained model.
+- Using the [SHAP EXPLAINER](https://github.com/slundberg/shap) toolkit to interpret the trained model that you can know
+how these features affect the car price.
 
-## The Dataset
+## Car Price Dataset
 
-This tutorial would use the [Car Features and MSRP](https://www.kaggle.com/CooperUnion/cardataset) as the demonstration dataset from kaggle.
+This tutorial would use the [Cars Dataset](https://www.kaggle.com/CooperUnion/cardataset) as the demonstration dataset from [kaggle](https://www.kaggle.com/).
 
-We have used feature engineering to preprocess the raw data and create new features. The feature engineering file is `carprice_preprocessing.py` in the current directory. Finally, We separated the dataset into the train/test dataset and got the `trainSet.csv` and `testSet.csv` files, which is used to train/predict our model. 
+The preprocessed data has already loaded in MySQL, and you can check the Python script [carprie_preprocessing.py](/doc/tutorial/didi/carprice_xgboost/carprie_preprocessing.py) to reproduce the preprocessing.
```

Comment:
"has already loaded in MySQL" => "has been loaded to MySQL" ?

Reply:
done.